cluster/src/main/java/com/hartwig/pipeline/tertiary/HmfToolCommandBuilder.java

```diff
@@ -36,6 +39,12 @@ public HmfToolCommandBuilder reference(final String referenceSample, final Strin
         return this;
     }
 
+    @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+    public HmfToolCommandBuilder panelBed(final Optional<String> panelBedLocation) {
+      //  arguments.addAll(panelBedLocation.stream().flatMap(l -> Stream.of("-panel_bed", l)).collect(Collectors.toList()));
```

Comment:
This is commented just because not all commands support it yet?

Reply:
yes